lib/webhdfs/client_v1.rb

```diff
@@ -60,6 +61,104 @@ def initialize(host='localhost', port=50070, username=nil, doas=nil, proxy_addre
       @kerberos = false
       @kerberos_keytab = nil
       @http_headers = http_headers
+
+      @jmx = nil
+    end
+
+    def self.simple(opts = {})
+      opts = {
+        :jmx => ENV['JMX'],
+        :host => ENV['DEFAULT_NAMENODE'] || 'localhost',
+        :port => 50070,
+        :kerberos => ENV['KERBEROS'] && true,
+        :kerberos_keytab => ENV['KERBEROS_KEYTAB'],
+
+        :retry_known_errors => false,
+        :retry_times => 1,
+        :retry_interval => 1,
+
+        :httpfs_mode => false,
+
+        :ssl => false,
+      }.merge(opts)
+
+      client = new
+
+      client.host = opts[:host]
```

Comment:
I definitely don't know ruby well enough to know if this is good/bad practice, but I think we could DRY out these assignments with something like 
```ruby
opts.keys.each do |key|
  m = "#{key}="
  client.send( m, opts[key] ) if client.respond_to?( m )
end
```
per [stackoverflow](https://stackoverflow.com/questions/3669801/dry-way-to-assign-hash-values-to-an-object)

Reply:
Sounds good to me 👍

Reply:
Implemented in bcdabf3